vars/fcosKolaTestIso.groovy

```diff
@@ -34,6 +34,7 @@ def call(params = [:]) {
             } else {
                 shwrap("cd ${cosaDir}  && kola testiso -S ${extraArgs} --output-dir tmp/kola-testiso-metal")
             }
```

Comment:
if we leave it like this and don't make it a scenario we probably want a comment here explaining what we are doing.